manifests/02_service.yaml

```diff
@@ -0,0 +1,20 @@
+# Expose operator metrics
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: openshift-authentication-operator-serving-cert
+  labels:
+    app: openshift-authentication-operator
+  name: metrics
```

Comment:
Prefer `openshift-authentication-operator` for consistency.

Reply:
I used `metrics` to be consistent with the other operators (`openshift-apiserver-operator`, `openshift-kube-scheduler-operator`, `openshift-controller-manager-operator`, etc). It's also preferred only namespaces have the `openshift-` prefix with names kept simple; I figure we can update the other stuff in a separate pr.

Reply:
OK.  I suggest changing the name of all resources in the OAuth server namespace from `openshift-authentication` to something like `oauth` or `integrated-oauth-server`.  Calling the service of the OAuth server `metrics` is incorrect, but you could create a new service with that name if you so desired (and maybe skip renaming the rest all together).